src/detectors/FHCAL/FHCAL.cc

```diff
@@ -113,7 +113,7 @@ extern "C" {
              "HcalEndcapPInsertTruthClusterAssociations"}, // edm4eic::MCRecoClusterParticleAssociation
             {
               .energyWeight = "log",
-              .sampFrac = 1.0,
+              .sampFrac = 0.0257,
```

Comment:
Needs to be updated for `HcalEndcapPInsertClusters` as well

Reply:
Truth clusters need to be changed too.

Reply:
Hmm... It appears I was confused about what "truth clusters" means... After looking through the code, it appears to  mean that truth information is used to determine which hits are in the cluster, but it doesn't use the truth energy of the hits.  Based on this, I reintroduced the sampFrac correction in the latest commit.